extension_version.txt

```diff
@@ -1 +1 @@
-published_qaextension_version=1.10.1
```

Comment:
what is this file used for?

Reply:
I think this was added as part of the QAExt pipeline automation.

Reply:
this file we need for publishing the extension using the automation pipeline. because when we upgrade the patch version we need to keep track of previous QAExtension version